i6_models/parts/conformer/convolution.py

```diff
@@ -85,3 +91,62 @@ def forward(self, tensor: torch.Tensor) -> torch.Tensor:
         tensor = self.pointwise_conv2(tensor)
 
         return self.dropout(tensor)
+
+
+@dataclass
+class ConformerConvolutionV2Config(ConformerConvolutionV1Config):
+    """
+    New attribute:
+        dropout_broadcast_axes: string of axes to which dropout is broadcast, e.g. "T" for broadcasting to the time axis
+                                setting to None to disable broadcasting
+    Allows even kernel size
```

Comment:
I don't understand this comment. You mean `ConformerConvolutionV2` can handle even kernel sizes? But how? I don't see that the code is different from earlier? Why is it supported now?